app/pulp/app/tasks/orphan.py

```diff
@@ -0,0 +1,14 @@
+from celery import shared_task
+
+from pulp.app import models
+from pulp.tasking.tasks import UserFacingTask
+
+
+@shared_task(base=UserFacingTask)
+def delete_all():
+    """
+    Delete all orphan content.
+    This task removes content from the filesystem as well.
+    """
+    orphans = models.Content.objects.exclude(id__in=models.RepositoryContent.objects.all())
```

Comment:
Here is query:
```
SELECT "pulp_app_content"."id", "pulp_app_content"."type" FROM "pulp_app_content"
    WHERE NOT ("pulp_app_content"."id" IN 
        (SELECT U0."content_id" FROM "pulp_app_repositorycontent" U0))
```
So I guess pagination is needed for the subquery. What paginator to use? We already have the DRF one but it requires `request`, iiuc. So should [Django paginator](https://docs.djangoproject.com/en/1.8/topics/pagination/#paginator-objects) be used in this case?